monai/apps/reconstruction/mri_dictionary.py

```diff
@@ -0,0 +1,285 @@
+from monai.transforms.croppad.dictionary import Cropd
+from monai.transforms.intensity.array import NormalizeIntensity
+from monai.transforms.transform import MapTransform, RandomizableTransform
+from monai.utils import FastMRIKeys
+from monai.utils.type_conversion import convert_to_tensor
+
+
+class RandomKspaceMaskd(RandomizableTransform, MapTransform):
+    """
+    Dictionary-based wrapper of :py:class:`monai.transforms.RandomKspaceMask`.
+
+    Args:
+        keys: keys of the corresponding items to be transformed.
+            See also: monai.transforms.MapTransform
+        center_fractions: Fraction of low-frequency columns to be retained.
+            If multiple values are provided, then one of these numbers is
+            chosen uniformly each time.
+        accelerations: Amount of under-sampling. This should have the
+            same length as center_fractions. If multiple values are provided,
+            then one of these is chosen uniformly each time.
+        spatial_dims: Number of spatial dims (e.g., it's 2 for a 2D data; it's
+            also 2 for psuedo-3D datasets like the fastMRI dataset).
+            The last spatial dim is selected for sampling. For the fastMRI
+            dataset, k-space has the form (...,num_slices,num_coils,H,W)
+            and sampling is done along W. For a general 3D data with the
+            shape (...,num_coils,H,W,D), sampling is done along D.
+        is_complex: if True, then the last dimension will be reserved
+            for real/imaginary parts.
+        allow_missing_keys: don't raise exception if key is missing.
+    """
+
+    backend = RandomKspaceMask.backend
+
+    def __init__(
+        self,
+        keys: KeysCollection,
+        center_fractions: Sequence[float],
+        accelerations: Sequence[float],
+        spatial_dims: int = 2,
+        is_complex: bool = True,
+        allow_missing_keys: bool = False,
+    ) -> None:
+        MapTransform.__init__(self, keys, allow_missing_keys)
+        self.masker = RandomKspaceMask(
+            center_fractions=center_fractions,
+            accelerations=accelerations,
+            spatial_dims=spatial_dims,
+            is_complex=is_complex,
+        )
+
+    def set_random_state(
+        self, seed: Optional[int] = None, state: Optional[np.random.RandomState] = None
+    ) -> "RandomKspaceMaskd":
+        super().set_random_state(seed, state)
+        self.masker.set_random_state(seed, state)
+        return self
+
+    def __call__(self, data: Mapping[Hashable, NdarrayOrTensor]) -> Dict[Hashable, Tensor]:
+        """
+        Args:
+            data: is a dictionary containing (key,value) pairs from the
+                loaded dataset
+
+        Returns:
+            the new data dictionary
+        """
+        d = dict(data)
+        for key in self.key_iterator(d):  # key is typically just "kspace"
+            d[key + "_masked"], d[key + "_masked_ifft"] = self.masker(d[key])
```

Comment:
Is it necessary to keep consistent random behavior for keys?
If yes, need to randomize in dict level and implement `randomize=False` in array level, refer to:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/transforms/intensity/dictionary.py#L214

Thanks.